sim/SPI_v3/components/Loopback.py

```diff
@@ -0,0 +1,51 @@
+'''
+==========================================================================
+Loopback.py
+==========================================================================
+This takes receives a packet from the SPIMinionAdapter module and sends it back the next cycle to the 
+SPIMinionAdapter module. Uses val/rdy microprotocol.
+
+Author: Dilan Lakhani
+    December 13, 2021
+
+'''
+
+from pymtl3 import *
+from pymtl3.stdlib.basic_rtl import RegisterFile, Mux, RegEnRst, RegEn, Adder, Incrementer, LeftLogicalShifter
```

Comment:
Maybe clean up the imports a bit? Seems that only `RegEnRst` is used

Reply:
Changed imports to only RegEnRst